### ISSUE TITLE
Ability to indicate an array in response parameters.

### ISSUE THREAD

**Issue by jeffutter** [zipmark/rspec_api_documentation#241]
I can't seem to find anywhere this is mentioned in the docs or tests. Is there any way to indicate that parameters are in an array of like objects?

For example, with the response

```json
{
  "users": [
    {
      "id": 1,
      "username": "foo"
    },
    {
      "id": 2,
      "username": "bar"
    }
  ]
}
```

would have response parameters like:

```
users[id]
users[username]
```

I would expect some way to output:

```
users[][id]
users[][username]
```

Am I missing something? Any thoughts?

**Comment by oestrich** [comment]
I do not believe that this is possible right now. This is the `response_field` DSL? I can't think of a nice way around this, I will be happy to look at a PR if this is still an issue.

My first thought is to allow you to pass an extra option to the response_field call specifying exactly what the return scoping/field will be, `:field => "users[][id]` perhaps. This might be cumbersome quick though.

**Comment by h55nick** [comment]
I know this is older but any updates around this? has anyone found a solution?

**Comment by kopylovvlad** [comment]
Has anyone found a solution? It is still actual 😓

**Comment by kopylovvlad** [comment]
I have found a solution: https://github.com/zipmark/rspec_api_documentation/issues/430
